sae_bench/evals/absorption/feature_absorption_calculator.py

```diff
@@ -150,19 +152,18 @@ def calculate_absorption(
         probe_direction: torch.Tensor,
         main_feature_ids: list[int],
         layer: int,
-        filter_prompts: bool = True,
+        filter_prompts: bool = False,
```

Comment:
nit: It seems like we never want to filter prompts anymore, and this shouldn't matter for the original absorption metric as `_is_full_absorption()` checks this condition as well. This option should be deleted and the `_filter_prompts()` method should be deleted as well.